metal/resource_metal_port_vlan_attachment_test.go

```diff
@@ -212,12 +212,12 @@ resource "metal_device_network_type" "test" {
 resource "metal_vlan" "test" {
   description = "test vlan"
   facility    = "dfw2"
```

Comment:
maybe we can subst the facility string literal here with `metal_device.test.deployed_facility`